src/contexts/index.ts

```diff
@@ -1,9 +1,25 @@
 /**
  * Contexts barrel export
- * Provides centralized exports for all React contexts
+ *
+ * DEPRECATED: This file now re-exports from the Zustand store for backward compatibility.
```

Comment:
Remove all deprecated content. Do not concern yourself with backward compatibility; migrate all usages forward to the intended implementation